applications/Cook Protocol.md

```diff
@@ -0,0 +1,151 @@
+https://github.com/CookFinance/cook-leverage  
+Price-based and time-based token distribution and liquidity mining:  
+https://github.com/CookFinance/cook-distribution-and-reward  
+
+### Team Code Repos
+* https://github.com/CookFinance
+
+### Team LinkedIn Profiles
+* https://www.linkedin.com/in/adrian-p-6564771b5/
+* https://www.linkedin.com/in/cage-chen-17a557107/
+* https://www.linkedin.com/in/jikun-liao-794b61162/
+* https://www.linkedin.com/in/kunpeng89/
+
+## Development Roadmap :nut_and_bolt: 
+
+### Overview
+* **Total Estimated Duration:** 2 months
+* **Full-time equivalent (FTE):**  2 FTE
+* **Total Costs:** 14k DAI
+
+### Milestone 1 Initialize Parachain & Frontend Implementation
+* **Estimated Duration:** 2 weeks
+* **FTE:**  2
+* **Costs:** 4k DAI
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 0a. | License | Apache 2.0 / MIT / Unlicense |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) spin up one of our Substrate nodes. Once the node is up, it will be possible to send test transactions that will show how the new functionality works. |
+| 0c. | Testing Guide | The code will have unit-test coverage (min. 70%) to ensure functionality and robustness. In the guide we will describe how to run these tests | 
+| 0d. | Article/Tutorial | We will write an article or tutorial that explains the work done as part of the grant. 
+| 1. | Frontend | We will implement frontend using React.js so that:  1. Users can connect to the polkadot wallet.  2. The fund manager can initialize a fund, and allocate assets via Polkaswap.  3. The investor can browse the fund list, check the performance details of a fund, and invest in a fund. |  
+| 2. | Substrate | Create a placeholder parachain which frontend can interact with. |    
+| 3. | Docker | We will provide a dockerfile to demonstrate the full functionality |
+
+### Milestone 2 Fund Investment & Management Implementation
+* **Estimated Duration:** 3 weeks
+* **FTE:**  2
+* **Costs:** 6k DAI
+
+| Number | Deliverable | Specification |
+| ------------- | ------------- | ------------- |
+| 0a. | License | Apache 2.0 / MIT / Unlicense |
+| 0b. | Documentation | We will provide both inline documentation of the code and a basic tutorial that explains how a user can (for example) spin up one of our Substrate nodes. Once the node is up, it will be possible to send test transactions that will show how the new functionality works. |
+| 0c. | Testing Guide | The code will have unit-test coverage (min. 70%) to ensure functionality and robustness. In the guide we will describe how to run these tests | 
+| 0d. | Article/Tutorial | We will write an article or tutorial that explains the work done as part of the grant. 
+| 1. | Fund management | We will implement the API for fund managers to create a fund, manage a fund and claim management fees. |  
```

Comment:
Are you developing a substrate module/pallet here or something else?

Reply:
We are going to develop a pallet. Here is the updated version of our application:
"Tech Stack - We will develop a custom pallet and a Substrate node that will work as Parathread of Polkadot. Hopefully in the future, we will be able to secure a Parachain Slot. We will use Rust language and the Substrate framework for the node and pallet development. Typescript and ReactJs for UI."